datastream-to-influxdb.py

```diff
@@ -105,13 +105,13 @@ def _update_start_end(self):
 
     def should_wait(self):
         now = datetime.datetime.utcnow()
-        self.log.debug("checking wait", wait=self.wait, retries=self.retries, hostname=self.hostname)
+        self.log.debug("checking wait", wait=self.wait, retries=self.retries, hostname=self.hostname, wait_time=self.wait_time, start=self.start.isoformat(), end=self.end.isoformat())
 
         if self.wait is not None:
             return now < self.wait_time
 
         # only query of we are at least 5 minutes from end time
-        return now > self.end and (now - self.end).seconds >= 300
+        return not(now > self.end and (now - self.end).seconds >= 300)
```

Comment:
Jeg f0ler på meg at denne kan skrives til noe enklere. Den er ikke umiddelbart enkelt å forstå.

Reply:
forslag settes pris på

Reply:
Hva med noe som `now >= self.end or ...` eller lage en variabel med et godt navn for den andre sjekken der?

Reply:
Bedre slik?